.github/workflows/ci.yaml

```diff
@@ -37,7 +37,8 @@ jobs:
     steps:
       - uses: actions/checkout@8e8c483db84b4bee98b60c0593521ed34d9990e8 # v6
       - uses: ./.github/actions/prepare
-      - run: pnpm build --noCheck
+      - run: pnpm build
+      - run: pnpm tsc -b
```

Comment:
[Cleanup] I think we can get away with just `build`? Locally it works for me on a fresh clone.

```suggestion
```